Resources/Prototypes/_White/Recipes/Lathes/devices.yml

```
@@ -8,6 +8,36 @@
+- type: latheRecipe
+  id: ClothingEyesNightVisionSecurityGoggles
+  result: ClothingEyesNightVisionSecurityGoggles
+  completetime: 2
+  materials:
+    Steel: 200
+    Glass: 100
+    Silver: 100
+    Gold: 100
+
+- type: latheRecipe
+  id: ClothingEyesNightVisionMedicalGoggles 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)